modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2637,10 +2659,13 @@ def broadcast_apply_full_axis(
         enumerate_partitions : bool, default: False
             Whether pass partition index into applied `func` or not.
             Note that `func` must be able to obtain `partition_idx` kwarg.
-        dtypes : list-like, default: None
+        dtypes : list-like, optional
```

Comment:
Is it possible to use `list-like` type everywhere or `pandas.Series` one? To be more consistent.